test/suites/dev/moonbase/test-orbiters/test-orbiters.ts

```diff
@@ -0,0 +1,155 @@
+        // some rounds without producing blocks
+        await jumpRounds(context, 6);
+
+        const afterOrbiterActiveRound = await context.polkadotJs().query.parachainStaking.round();
+        const afterOrbiterActiveOrbiter = await context
+          .polkadotJs()
+          .query.moonbeamOrbiters.orbiterPerRound(
+            afterOrbiterActiveRound.current,
+            orbiterPool.address
+          );
+        expect(afterOrbiterActiveOrbiter.toString()).toEqual(orbiter.address);
+
+        const notifyInactiveOrbiter = await context.createBlock(
+          context
+            .polkadotJs()
+            .tx.parachainStaking.notifyInactiveCollator(orbiter.address)
+            .signAsync(alith),
+          { allowFailures: true }
+        );
+
+        expect(notifyInactiveOrbiter.result!.successful).toEqual(false);
+        expect(notifyInactiveOrbiter.result!.error!.name).toEqual("CannotBeNotifiedAsInactive");
+
+        // Call to mark an orbiterPool that has not produced blocks offline
+        // should succeed but it should be an noop (pool will still be active) if there are is an
+        // active orbiter in the pool
+        await context.createBlock(
+          context
+            .polkadotJs()
+            .tx.parachainStaking.notifyInactiveCollator(orbiterPool.address)
+            .signAsync(alith),
+          { allowFailures: false }
+        );
+
+        const afterNoopNotifyInfo = await context
+          .polkadotJs()
+          .query.parachainStaking.candidateInfo(orbiterPool.address);
+        expect(afterNoopNotifyInfo.unwrap().status.isActive).toBe(true);
+
+        await context.createBlock(
+          context
+            .polkadotJs()
+            .tx.moonbeamOrbiters.collatorRemoveOrbiter(orbiter.address)
+            .signAsync(orbiterPool),
+          { allowFailures: false }
+        );
+
+        // Advance rounds so that the orbiter gets rotated out of the pool
+        // and the OrbiterPerRoundEntry is cleared
+        await jumpRounds(context, 6);
+
+        // Marking the orbiter pool without active orbiters should
+        // make the orbiter pool idle
+        await context.createBlock(
+          context
+            .polkadotJs()
+            .tx.parachainStaking.notifyInactiveCollator(orbiterPool.address)
+            .signAsync(alith),
+          { allowFailures: false }
+        );
+
+        const afterRemoveOrbiterRound = await context.polkadotJs().query.parachainStaking.round();
+        const afterRemoveOrbiterOrbiter = await context
+          .polkadotJs()
+          .query.moonbeamOrbiters.orbiterPerRound(
+            afterRemoveOrbiterRound.current,
+            orbiterPool.address
+          );
+        expect(afterRemoveOrbiterOrbiter.isNone).toBe(true);
+
+        const afterOrbiterRemoveInfo = await context
+          .polkadotJs()
+          .query.parachainStaking.candidateInfo(orbiterPool.address);
+        expect(afterOrbiterRemoveInfo.unwrap().status.isIdle).toBe(true);
```

Comment:
After this, you could also check that the orbiter pool went out of the `CandidatePool`.

Reply:
Added